source/transfers/contracts/TransferHandlerRegistry.sol

```diff
@@ -0,0 +1,30 @@
+pragma solidity 0.5.12;
+
+import "./interfaces/ITransferHandler.sol";
+
+/**
+  * @title TransferHandlerRegistry: holds registry of contract to
+  * facilitate token transfers
+  */
+contract TransferHandlerRegistry {
+
+  event AddTransferHandler(
+    bytes4 kind,
```

Comment:
Should either of these be indexed?

Reply:
This event won't be used frequently as well as one can check via the transferHandlers public mapping.Thus, I did not add the index. However, it's low cost, there's no reason not too.